packages/contracts/test/SignupAndPost.test.ts

```diff
@@ -68,7 +68,7 @@ describe('Unirep App', function () {
             userState.stop()
         })
 
-        it('revert when user sign up multiple times', async function () {
+        it('revert when user reuse the same signup proof', async function () {
```

Comment:
it should be `revert when user sign up multiple times` (the original one)

Reply:
This is modified.